clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/ConsoleLogger.java

```diff
@@ -3,15 +3,23 @@
 
 package com.microsoft.aspnet.signalr;
 
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class ConsoleLogger implements Logger {
     private LogLevel logLevel;
+    private DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'");
```

Comment:
I think we want to use `Z` without quotes. If we use quotes, it's a literal `Z`, but if we don't use quotes it expands to the time zone in the correct format. So for Redmond you'd get `2018-08-10T12:54-0800`. If we know for sure it's always UTC, we can use `Z`, but I actually think logs should be in local time anyway.